### PR TITLE
Docker + cli + config updates

### PR DESCRIPTION
**Pull request by frostbyte73**

### CODE REVIEW

File: service/cmd/server/record.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+	"github.com/urfave/cli/v2"
+	"google.golang.org/protobuf/proto"
+
+	"github.com/livekit/livekit-recorder/service/pkg/config"
+	"github.com/livekit/livekit-recorder/service/pkg/logger"
+	"github.com/livekit/livekit-recorder/service/pkg/service"
+	livekit "github.com/livekit/livekit-recorder/service/proto"
+)
+
+// Acts as a livekit server against a recorder-service docker container with shared redis
```
Comment (davidzhao) [inline service/cmd/server/record.go:19]: who uses these functions? are these for testing?

File: service/README.md [hunk 2]
```diff
@@ -12,36 +12,36 @@ A single instance can record one room at a time.
 ## Config
 
 The only required field is redis address. This must be the same redis address used by your LiveKit server.  
-If you want to use templates without supplying your own tokens, `apiKey` and 
-`apiSecret` are also required.
+If you want to use templates without supplying your own tokens, `api_key` and 
+`api_secret` are also required.
 
 ```yaml
 {
```
[the remark this answers is not in the document]
Comment (frostbyte73) [reply service/README.md:19]: json for the recorder, yaml for the service

File: service/cmd/server/record.go (hunk 1 above)
Comment (frostbyte73) [reply service/cmd/server/record.go:19]: yeah, they're mostly for testing. But could be used for one-off recordings

File: recorder/Dockerfile [hunk 3]
```diff
@@ -16,9 +16,10 @@ RUN apt-get install -y ffmpeg
 WORKDIR /app
 COPY package.json package-lock.json tsconfig.json ./
 COPY src ./src
-RUN npm install \
-    && npm install typescript \
-    && npm install -g ts-node
+RUN npm install
+
+# Silence error about livekit-server-sdk protos
+RUN npx tsc src/*.ts; exit 0
```
Comment (davidzhao) [inline recorder/Dockerfile:22]: weird you are seeing this error. I tried it and it shows up too.. not sure what causes it.

**Review by davidzhao — APPROVED** [review]

**Pull request merged**